corehq/apps/hqwebapp/static/hqwebapp/js/lib/bootstrap-tab-hashes.js

```diff
@@ -1,7 +1,9 @@
-// https://gist.github.com/josheinstein/5586469
```

Comment:
This library is used only in case details so I modified it instead of creating new one.